src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget/StakingInput/StakingSliderMessages/RequiredStakeMessage.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import classNames from 'classnames';
+import Decimal from 'decimal.js';
+
+import Numeral from '~shared/Numeral';
+import { SLIDER_AMOUNT_KEY } from '../StakingInput';
+
+import styles from './RequiredStakeMessage.css';
+
+const displayName =
+  'common.ActionDetailsPage.DefaultMotion.StakingWidget.RequiredStakeMessage';
+
+const MSG = defineMessages({
+  requiredStake: {
+    id: `${displayName}.requiredStake`,
+    defaultMessage: ` ({stakePercentage}% of required)`,
+  },
+});
+
+const RequiredStakeMessage = () => {
+  const { watch } = useFormContext();
+  const sliderAmount = watch(SLIDER_AMOUNT_KEY);
+  const isUnderThreshold = sliderAmount < 10;
+  const isOverThreshold = !isUnderThreshold;
+  const nativeTokenDecimals = 18;
+  const nativeTokenSymbol = 'WILL';
+  const stake =
+    sliderAmount === 0
+      ? 0
+      : new Decimal('1000000000000000000').div(sliderAmount);
```

Comment:
If this is test data, disregard this, otherwise, since I assume this is related to the number of decimals the native token has, it needs to be set accordingly